go_api/cmd/initdb.go

```diff
@@ -0,0 +1,31 @@
+package cmd
```

Comment:
```suggestion
package main
```
惜しい！
ここは`package main`で大丈夫です。